upstairs/src/lib.rs

```diff
@@ -3436,6 +3435,11 @@ impl Downstairs {
             }
         }
 
+        info!(
+            self.log,
+            "[{}] changed {} to fault skipped", client_id, number_jobs_skipped
```

Comment:
Maybe: `Changed {} jobs to fault skipped`
Otherwise, I'll think {} is the job id :)

Reply:
agreed, a7eaf8f